cualid/label.py

```diff
@@ -26,7 +26,7 @@ def get_barcodes(input,
                  x_start=1.9,
                  y_start=257.2):
 
-    ids = [e.strip().split('\t')[1] for e in input]
+    ids = [e.strip().split('\t')[0] for e in input]
```

Comment:
Yes, I agree, however this change only make sense in the context of other changes, which need to be added to this pull request